modules/_node-scripts/format/format.mjs

```diff
@@ -230,6 +250,14 @@ export default async function format(fix, filesToFormat = undefined) {
 		summary += fixedFiles.map((file) => `  · ${file}`).join('\n');
 	}
 
+	if (suppressCurrentErrors) {
+		await fs.writeFile(
+			path.join(import.meta.dirname, 'silencedErrors.txt'),
```

Comment:
Are devs supposed to commit this file back if they supress current errors?

Reply:
Devs shouldn't be suppressing any new errors. Ideally only FI would be run this command that generates the supressions